src/main/java/pingis/controllers/MiscController.java

```diff
@@ -0,0 +1,18 @@
+package pingis.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class MiscController {
+
+    @RequestMapping("/")
+    public String index() {
+        return "index";
+    }
+
+    @RequestMapping("/sandbox")
```

Comment:
I would also be a bit hesitant to include anything so vaguely named, maybe rename to something like `IndexController`?